gossip-base/src/main/java/org/apache/gossip/event/GossipListener.java

```diff
@@ -21,4 +21,7 @@
 
 public interface GossipListener {
   void gossipEvent(Member member, GossipState state);
+
+  /* this listener's name */
+  String name();
```

Comment:
The user creating the Listener can decided to use a class with a property like name if they wish, but having two methods here removes the ability to use lambda's which I believe makes them less "fun"